namui/src/getGcCanvasKitPackage.ts

```diff
@@ -45,3 +52,14 @@ function makeItGc<T extends EmbindObject<any>>(
 
   return gcConstructor as any;
 }
+
+function makeItGcAlt<T extends EmbindObject<any>, P extends any[]>(
+  constructor: (...params: P) => T,
+  garbages: EmbindObject<any>[],
+) {
+  return function (...params: P): T {
+    const shader = constructor(...params);
+    garbages.push(shader);
+    return shader;
+  };
+}
```

Comment:
### Notice for Engine Developers
Create a new GCMaker function for `CanvasKit.Shader.MakeLinearGradient`

### Why new function
`makeItGc()` works with constructor like `new ()`. But `CanvasKit.Shader.MakeLinearGradient` is not like that.

### What I want to ask the reviewer

1. Is it the right way?
2. Is it the right name?